vault/logical_system_paths.go

```diff
@@ -435,6 +545,55 @@ func (b *SystemBackend) rekeyPaths() []*framework.Path {
 
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.UpdateOperation: &framework.PathOperation{
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields: map[string]*framework.FieldSchema{
+								"nounce": {
+									Type:     framework.TypeString,
+									Required: true,
+								},
+								"complete": {
+									Type: framework.TypeBool,
+								},
+								"started": {
+									Type: framework.TypeString,
+								},
+								"t": {
+									Type: framework.TypeInt,
+								},
+								"n": {
+									Type: framework.TypeInt,
+								},
```

Comment:
I'm not sure where these fields are defined, but should some of them be marked as required?

Reply:
So actually no only the three I have, this endpoint is fairly jank as it can return two totally different structures. more reading here: https://github.com/hashicorp/vault/blob/b741fa893d775a252c5e518c015aa1c94e30cc63/http/sys_rekey.go#L198-L220

Reply:
If the API is unstable (can return two different structures), it might be better to just not specify the fields at all. Not really sure, what do you think?

Reply:
I think it's more helpful to return all possible fields than nothing at all.

Reply:
so responses is `map[int][]framework.Response`
why not have
```go
http.StatusOK: {
    {
        Description: "returned if ...",
        Fields: ...
    },
    {
        Description: "returned if ...",
        Fields: ...
    }
}
```